### PR TITLE
Implement classic lighting mode

### PR DESCRIPTION
**Pull request by hasenbanck**
This mode aims to provide a lightning mode that is closer to the original. The default is still the enhanced lightning mode.

### CODE REVIEW

**Comment by hasenbanck** [comment]
I eyeballed the intensity of the point light inside the light map, and a factor of 10.0 looks kida right. We still have the natural falloff, so it also still looks fine in the enhanced mode. This PR is now in a state I would like to merge it.

File: korangar/src/graphics/passes/forward/shader/entity.wgsl [hunk 1]
```diff
@@ -205,16 +205,40 @@ fn fs_main(input: VertexOutput) -> FragmentOutput {
...
+    let light_contributions = saturate(ambient_light_contribution + directional_light_contribution + point_light_contribution);
+    var color = base_color.rgb * light_contributions;
+
+    if (global_uniforms.enhanced_lightning == 0) {
+        color = color_balance(color, -0.01, 0.0, 0.0);
     }
 
     var output: FragmentOutput;
-    output.fragment_color = vec4<f32>(final_color, final_alpha);
+    output.fragment_color = vec4<f32>(color, alpha_channel);
```
Comment (kokosha) [inline korangar/src/graphics/passes/forward/shader/entity.wgsl:222]: Remember to multiply alpha_channel by input.color.a

File: korangar/src/graphics/passes/forward/shader/entity_bindless.wgsl [hunk 2]
```diff
@@ -207,16 +207,40 @@ fn fs_main(input: VertexOutput) -> FragmentOutput {
...
+    let light_contributions = saturate(ambient_light_contribution + directional_light_contribution + point_light_contribution);
+    var color = base_color.rgb * light_contributions;
+
+    if (global_uniforms.enhanced_lightning == 0) {
+        color = color_balance(color, -0.01, 0.0, 0.0);
     }
 
     var output: FragmentOutput;
-    output.fragment_color = vec4<f32>(final_color, final_alpha);
+    output.fragment_color = vec4<f32>(color, alpha_channel);
```
Comment (kokosha) [inline korangar/src/graphics/passes/forward/shader/entity_bindless.wgsl:224]: The same here

File: korangar/src/graphics/passes/forward/shader/entity.wgsl (hunk 1 above)
Comment (hasenbanck) [reply korangar/src/graphics/passes/forward/shader/entity.wgsl:222]: Nice catch!

File: korangar/src/graphics/passes/forward/shader/entity_bindless.wgsl (hunk 2 above)
Comment (hasenbanck) [reply korangar/src/graphics/passes/forward/shader/entity_bindless.wgsl:224]: Nice catch!

**Pull request merged**